### PR TITLE
⚾️👴 Base64 decode

### PR DESCRIPTION
**Pull request by Gilad-Gur-Andelman**

### CODE REVIEW

File: src/main-process/proxy/encoding.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import isBase64 from 'is-base64';
+
+export const getEncoding = (text: string): 'base64' | undefined => {
```
Comment (onniaa) [inline src/main-process/proxy/encoding.ts:3]: classic enum
```ts
enum Encoding {
  BASE_64 = 'base64',
}
```

**Comment by onniaa** [comment]
Very cool, works great for me 💪

**Comment by Gilad-Gur-Andelman** [comment]
Alt Text

**Comment by idoco** [comment]
Works great for me

**Review by yigaldviri — APPROVED** [review]

**Comment by Gilad-Gur-Andelman** [comment]
## 🏗 Build Artifacts are available here👇 

### 1. [🍏 .dmg](https://output.circle-artifacts.com/output/job/7d9b9098-3156-42eb-b48b-efbc412b312a/artifacts/0/out/make/Loadmill-3.0.4-x64.dmg)

### 2. [🤐 .zip](https://output.circle-artifacts.com/output/job/7d9b9098-3156-42eb-b48b-efbc412b312a/artifacts/0/out/make/zip/darwin/x64/Loadmill-darwin-x64-3.0.4.zip)

**Pull request merged**